mc-sema/bin_descend/get_cfg.py

```diff
@@ -109,6 +109,7 @@ def xrange(begin, end=None, step=1):
 def DEBUG(s):
     if _DEBUG:
         syslog.syslog(str(s))
+        sys.stdout.write(str(s))
```

Comment:
Write to `stderr`, or instead, have a command-line flag that can take a file name. If present, `open` it and use it, otherwise use `sys.stderr`.